scripts/ldap/replication/setupReplicationScript.sh

```diff
@@ -76,16 +76,31 @@ getServerIdFromHostFile()
     done < "$host_list"
 }
 
+op=$(ldapsearch -w seagate -x -D cn=admin,cn=config -b cn=config "olcServerID")
```

Comment:
it seems the hardcode rootdn password being used here. Please ask for the rootdn password on cmdline.